### PR TITLE
test: calendar file in invitation email

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

---

- Adds test if invitation email contains calendar file.
- With briefly checked contents.

### CODE REVIEW

File: cypress/integration/dashboard/events.js [hunk 1]
```diff
@@ -161,6 +161,38 @@ describe('events dashboard', () => {
...
   }
 
+  it('invitation email should include calendar file', () => {
+    cy.visit('/dashboard/events/1');
+
+    cy.findByRole('button', { name: 'Email Attendees' }).click();
+
+    // try to make sure there will be recipient
+    cy.findByLabelText('Waitlist').click({ force: true });
+    cy.findByLabelText('Cancelled').click({ force: true });
```
Comment (ojeytonwilliams) [inline cypress/integration/dashboard/events.js:171]: Do we need to force, here? It's a bit of an indication that something's not quite right.

**Review by ojeytonwilliams — COMMENTED** [review]
This looks nice.  It's not a big deal, but I was curious about the forcing:

File: cypress/integration/dashboard/events.js (hunk 1 above)
Comment (gikf) [reply cypress/integration/dashboard/events.js:171]: This is the matter in few other tests in `events.js` as well, for some reason cypress detects those elements as hidden by another element, and without forcing, it doesn't work. I didn't dig too deep to find what's up with that.

Comment (ojeytonwilliams) [reply cypress/integration/dashboard/events.js:171]: Understood. I'll add that to my backlog, then.

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**